src/jit/lsraarm.cpp

```diff
@@ -41,6 +41,9 @@ int LinearScan::BuildLclHeap(GenTree* tree)
     //   >4 ptr words               Yes             1
     //   Non-const                  Yes             1
     //   Non-const                  No              1
+    //
+    // If the outgoing argument space is too large to encode in an "add/sub sp, icon"
+    // instruction, we also need a temp (which can be shared).
```

Comment:
I'm not sure what it means to be shared. The internal registers are only live within the code generation for this single node.

Reply:
This single node generates a lot of code, and there are multiple conditions that require a temp register. This is just to say that there we can use the same temp register if there are multiple conditions that require one. I'll clarify the comment.

Reply:
Great, thanks.